Manito/Manito/Screens/Interaction/OpenManittoViewController.swift

```diff
@@ -144,20 +146,20 @@ final class OpenManittoViewController: BaseViewController {
     private func presentPopupViewController() {
         let storyboard = UIStoryboard(name: "Interaction", bundle: nil)
         guard let viewController = storyboard.instantiateViewController(withIdentifier: OpenManittoPopupViewController.className) as? OpenManittoPopupViewController else { return }
-        viewController.manittoText = manitto
+        viewController.manittoNickname = manitto
```

Comment:
p2;

```suggestion
        viewController.manittoNickname = self.manitto
```

요기도 빠져있군요.. 하하